.travis.yml

```diff
@@ -1,10 +1,10 @@
 language: elixir
 
 elixir:
-  - 1.6.0
+  - 1.8.2
```

Comment:
I would have preferred these bumps in (i) a single PR and (ii) one by one so we could rollback quickly.

I'm super sensitive to changes in this because (i) is on the critical path and (ii) nobody (apart you, of course) knows how get dirty with it.

Reply:
These bumps are needed to use the official New Relic agent for Elixir, `1.8.X` is the minimum version. They are at the same PR as the bumps because they are tied with the functionally being added, if we rollback the version, we need to rollback the instrumentation as well and there is no point on rollback the functionally and keep the version updated as we plan to change this code as least as possible.

I agree with you about the code changes. I had a ton of problems making this work, it's really "old code", lots of deprecations, etc.. etc.. we need to replace this asap.